.github/workflows/release.yml

```diff
@@ -3,7 +3,7 @@ name: Main
 on:
     push:
         tags:
-            - "*"
+            - "v1.*"
```

Comment:
Why not `"*"`?

Reply:
If something goes wrong, we don't accidentally want to publish major versions or other tags

Reply:
Cool.